### PR TITLE
FUNCAKE-89 Link Update About to PA Digital About

### PR DESCRIPTION
**Pull request by nomadicoder**
- Create about link
- Remove splash page text

### CODE REVIEW

**Comment by nomadicoder** [comment]
@bibliotechy Ready for review.

File: app/helpers/funnelcake_helper.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module FunnelcakeHelper
+
+  def about_url()
+    config = YAML.load_file(File.expand_path("#{Rails.root}/config/funnelcake.yml", __FILE__))
```
Comment (bibliotechy) [inline app/helpers/funnelcake_helper.rb]: This should be done with `config_for` in an initializer or `config/application.rb`, like in [cob](https://github.com/tulibraries/tul_cob/blob/qa/config/application.rb#L24). Doing it here means we do IO every time the helper is called. Doing it in the initializer or application.rb means it only happens when the application boots up.

Comment (nomadicoder) [reply app/helpers/funnelcake_helper.rb]: Done.

**Comment by bibliotechy** [comment]
@nomadicoder Don't think we need to keep that config out of version control, and I'd rather not have to add a step to the playbook for it. So just leave `funnel_cake.yml` in git and remove the example

**Pull request merged**